src/main/scala/beam/sim/BeamServices.scala

```diff
@@ -112,9 +115,16 @@ object BeamServices {
     new TAZTreeMap(tazQuadTree)
   }
 
-  def getTazTreeMap(file: String): TAZTreeMap = {
-    Try(TAZTreeMap.fromCsv(file)).getOrElse {
-      BeamServices.defaultTazTreeMap
+  def getTazTreeMap(filePath: String): TAZTreeMap = {
+    try {
+      TAZTreeMap.fromCsv(filePath)
+    } catch {
+      case fe: FileNotFoundException =>
+        logger.error("No TAZ file found at given path : %s" format filePath,fe)
```

Comment:
Nice. I hadn't paid close enough attention that this can't do the typical formatting since throwable is blocking it.